### ISSUE TITLE
No V-Sync. Gameplay dont't feel smooth.

### ISSUE THREAD

**Issue by lionnesquaer** [Nukem9/dlssg-to-fsr3#6]
Greetings!

I just try it in Cyberpunk 2077. It looks like work, but v-sync is disabled and greyed out. It most probably hardlock, becouse game think i using DLSS Frame Generation.
But like i know AMD FSR 3.0 Frame Generation NEED V-Sync for smooth gameplay. Otherwise i don't know reason of micro stutters.

**Comment by ChaosPaladin** [comment]
Did you try enable vsync override with nvidia control panel?

**Comment by lionnesquaer** [comment]
@ChaosPaladin Yes, i figured out trying it already. It became smoother. Also disabled RTSS. It seem all work fine!
